disttask/loaddata/dispatcher.go

```diff
@@ -43,6 +45,9 @@ func (*FlowHandle) ProcessNormalFlow(ctx context.Context, _ dispatcher.TaskHandl
 
 	switch gTask.Step {
 	case Import:
+		if err := h.postProcess(ctx, handle, gTask); err != nil {
```

Comment:
checksum might be quite long, will block framework dispatcher

Reply:
This is okay, the dispatcher can handle long-term tasks.